docs/concepts/storage/volumes.md

```diff
@@ -849,6 +841,19 @@ request a certain amount of space using a [resource](/docs/user-guide/compute-re
 specification, and to select the type of media to use, for clusters that have
 several media types.
 
+## Out-of-Tree Volume Plugins
+In addition to the previously listed volume types, storage vendors may create
+custom plugins without adding it to the Kubernetes repository. This can be
+achieved by using the `FlexVolume` plugin.
+
+`FlexVolume` enables users to mount vendor volumes into a pod. The vendor plugin
+is implemented using a driver, an executable supporting a list of volume commands
+defined by the `FlexVolume` API. Drivers must be installed in a pre-defined
+volume plugin path on each node. This is an alpha feature and may change in future.
+
+More details are in [here](https://github.com/kubernetes/examples/tree/{{page.githubbranch}}/staging/volumes/flexvolume/README.md).
```

Comment:
Looks like this URL should be updated to `https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md`.

Also, change the sentence wording to:

More details can be found [here](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md).

Reply:
ACK